README.md

```diff
@@ -368,11 +369,97 @@ ser_struct! {
 
 ```
 
+### Async support
+It is possible to use `async/await` http client, for that it is necessary to enable feature `async` in transistor, `transistor = { version = "1.1.0", features = ["async"] }`. With this feature enabled the `HttpClient` will use `reqwest::Client` instead of `reqwest::blocking::Client`. The default async runtime for `reqwest::Client` is `tokio`, so it is good to have `tokio` with feature `macros`, as well as `futures`, in your `Cargo.toml`:
+
+```toml
+futures = {version = "0.3.5" }
+tokio = {version = "0.2.22", features = ["macros"] }
+```
+
+An async query example can be found below:
+
+```rust
+use tokio::prelude::*;
+use transistor::client::Crux;
+use transistor::edn_rs::{ser_struct, Serialize};
+use transistor::types::http::Action;
+use transistor::types::{
+    error::CruxError,
+    {query::Query, CruxId},
+};
+
+#[tokio::main]
+async fn main() {
+    let crux = Database {
+        crux__db___id: CruxId::new("crux"),
+        name: "Crux Datalog".to_string(),
+        is_sql: false,
+    };
+
+    let psql = Database {
+        crux__db___id: CruxId::new("postgres"),
+        name: "Postgres".to_string(),
+        is_sql: true,
+    };
+
+    let mysql = Database {
+        crux__db___id: CruxId::new("mysql"),
+        name: "MySQL".to_string(),
+        is_sql: true,
+    };
+
+    let client = Crux::new("localhost", "3000").http_client();
+    let action1 = Action::Put(crux.serialize(), None);
+    let action2 = Action::Put(psql.serialize(), None);
+    let action3 = Action::Put(mysql.serialize(), None);
+
+    let _ = client.tx_log(vec![action1, action2, action3]).await.await;
```

Comment:
Is there any way we can use only one `await` call? I don't know async in Rust :confused:

Reply:
It should, I am trying to figure it out, but did not find the problem yet. Basically we call an async function inside an async block. So because of this 2 awaits

Reply:
Ok, for the first version of this I think we can go like this